### ISSUE TITLE
Add a refresh button to the filters page

### ISSUE THREAD

**Issue by rehandalal** [mozilla-extensions/normandy-devtools#172]
It would be nice to re-evaluate filter expressions to check if something has changed so we should have a button that lets us refresh the value without needing to update the editor.

**Comment by OlaoluwaM** [comment]
Hey @rehandalal is this issue still up, could you walk me through it. I would love to contribute

**Comment by rehandalal** [comment]
Yup, still up for grabs. Unfortunately I don't have the bandwidth to walk you through contribution right now, sorry. I'm happy to review any PRs though.

**Comment by rehandalal** [comment]
I will also add this is an internal tool that may not be the most easy place for community contribution. If you are looking for ways to contribute to Mozilla or an open source project, I would suggest checking out https://whatcanidoformozilla.org for ideas.
